Dockerfile

```diff
@@ -40,14 +39,12 @@ FROM --platform=$BUILDPLATFORM ${GOLANG_IMAGE} AS base
 COPY --from=xx / /
 RUN echo 'Binary::apt::APT::Keep-Downloaded-Packages "true";' > /etc/apt/apt.conf.d/keep-cache
 ARG APT_MIRROR
-RUN test -n "$APT_MIRROR" && sed -ri "s#(httpredir|deb|security).debian.org#${APT_MIRROR}#g" /etc/apt/sources.list || true
-ARG DEBIAN_FRONTEND
+RUN test -n "$APT_MIRROR" && sed -ri "s#(httpredir|deb|security).debian.org#${APT_MIRROR}#g" /etc/apt/sources.list.d/debian.sources || true
```

Comment:
Still of the opinion that we should just drop the `APT_MIRROR` code, but non-blocking. :innocent:

Reply:
Yeah, not strongly against, mostly curious if there's still users for it (I somewhat assume people behind the great firewall would already have things in place for such things though)

I'll keep that as a follow-up PR, also to make such a change more visible (instead of implicit change in an update)

Reply:
- opened https://github.com/moby/moby/pull/46884 for this